src/components/EstimatedLifecycleCost/index.tsx

```diff
@@ -19,89 +19,103 @@ import './index.scss';
 };
-const Phase = ({ formikKey, index, values }: PhaseProps) => {
+const Phase = ({ formikKey, year, index, values, errors = [] }: PhaseProps) => {
+  const phaseError = errors[index] || {};
   return (
-    <FieldArray name={formikKey}>
+    <FieldArray name={`${formikKey}.year${year}`}>
       {arrayHelpers => (
         <div className="est-lifecycle-cost__phase-cost-wrapper">
           <FieldGroup
-            className="est-lifecycle-cost__field-group"
             scrollElement="somethingGoesHere"
-            error={false}
+            error={phaseError.phase || phaseError.cost}
           >
-            <FieldErrorMsg />
-            <FieldErrorMsg />
-            <fieldset className="usa-fieldset est-lifecycle-cost__phase-fieldset">
-              <div className=" est-lifecycle-cost__phase-field-wrapper">
-                <legend className={classnames('usa-label', 'margin-bottom-1')}>
-                  Phase
-                </legend>
-                <div>
-                  <Field
-                    as={RadioField}
-                    checked={values.phase === 'Initiate'}
-                    id={`BusinessCase-${formikKey}.Phase${index}.initiate`}
-                    name={`${formikKey}.${index}.phase`}
-                    label="Initiate"
-                    value="Initiate"
-                    inline
-                  />
+            {phaseError.phase && (
+              <FieldErrorMsg>{phaseError.phase}</FieldErrorMsg>
+            )}
+            {phaseError.cost && (
+              <FieldErrorMsg>{phaseError.cost}</FieldErrorMsg>
+            )}
+            <div className="est-lifecycle-cost__field-group">
+              <fieldset className="usa-fieldset est-lifecycle-cost__phase-fieldset">
+                <div className=" est-lifecycle-cost__phase-field-wrapper">
+                  <legend
+                    className={classnames('usa-label', 'margin-bottom-1')}
+                    aria-label={`Year ${year} Phase ${index + 1} Phase Type`}
+                  >
+                    Phase
+                  </legend>
+                  <div>
+                    <Field
+                      as={RadioField}
+                      checked={values.phase === 'Initiate'}
+                      id={`BusinessCase-${formikKey}.Year${year}.Phase${index}.initiate`}
+                      name={`${formikKey}.year${year}.${index}.phase`}
+                      label="Initiate"
+                      value="Initiate"
+                      inline
+                    />
 
-                  <Field
-                    as={RadioField}
-                    checked={values.phase === 'Operations & Maintanence'}
-                    id={`BusinessCase-${formikKey}.Phase${index}.opsMaintanence`}
-                    name={`${formikKey}.${index}.phase`}
-                    label="Operations & Maintanence"
-                    value="Operations & Maintanence"
-                    inline
-                  />
+                    <Field
```

Comment:
In testing the phase fields out, I noticed I could have more than 2 phases per year. Should there only be two since there is operations and maintenance or initiate?

Reply:
That's a good question. I'm not sure. I didn't see any product requirements, so I assumed that a user can add as many as they want.